pytorch_lightning/utilities/argparse_utils.py

```diff
@@ -200,13 +200,36 @@ def add_argparse_args(cls, parent_parser: ArgumentParser) -> ArgumentParser:
             dest=arg,
             default=arg_default,
             type=use_type,
-            help='autogenerated by pl.Trainer',
+            help=args_help.get(arg),
             **arg_kwargs,
         )
 
     return parser
 
 
+def parse_args_from_docstring(docstring: str) -> Dict[str, str]:
+    arg_block_indent = None
+    current_arg = None
+    parsed = {}
+    for line in docstring.split("\n"):
+        stripped = line.lstrip()
+        if not stripped:
+            continue
+        line_indent = len(line) - len(stripped)
+        if stripped.startswith('Args:'):
```

Comment:
```suggestion
        if stripped.startswith(('Args:', 'Arguments:', 'Parameters:')):
```
to make it a bit general, we could include all aliases of "Args", i.e., "Arguments", "Parameters". 
See https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections